src/foremast/app/create_app.py

```diff
@@ -82,7 +82,7 @@ def get_accounts(self, provider='aws'):
 
         return filtered_accounts
 
-    def create_app(self):
+    def create_app(self, pipeline_configs):
```

Comment:
I think this should be a keyword argument, will be more consistent with other interfaces we have.

Also, update the doc string to include it